ScreenBoom/ScreenBoom/ViewModel/EventViewModel.swift

```diff
@@ -55,23 +55,21 @@ class EventViewModel: NSObject {
   
   
   // add event method takes event and completion to insert the event in the firbase
-  //Database and return result (error or firebasedatareferense
-  func addEvent(event: Event, oldEventCode: String? = "", completion:(@escaping(Result<String>) -> Void)) {
-    var eventCode = ""
-    if let oldEventCode = oldEventCode {
-      eventCode = oldEventCode
-    } else {
-      eventCode = String.random()
-    }
+  //Database and return result (error or eventCode
```

Comment:
this comment doesnt make sense. rewrite to explain whats happening in this method